image/templates/helm/stackrox-central/templates/01-central-12-central-db.yaml

```diff
@@ -31,6 +31,7 @@ spec:
           {{- $_ := set $annotations "openshift.io/required-scc" "nonroot-v2" -}}
         {{- end }}
         {{- include "srox.podAnnotations" (list . "deployment" "central-db" $annotations) | nindent 8 }}
+        checksum/config: {{- include (print $.Chart.Name "/templates/01-central-08-db-configmap.yaml") . | sha256sum | indent 1 }}
```

Comment:
Added this to restart the database when the config map settings are updated so they take effect.  Open to suggestions.

Reply:
I am trying to understand how include parses the input the function. According to the [documentation](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-include-function) the first parameter is a function, the second the input. `$.Chart.Name` is not a function, I would have expected `toYaml` instead to pipe the content of the ConfigMap to the next stage.
How does this work? Am I missing something? :thinking:

Reply:
@SimonBaeumer 
`print` is the function (I think).  If you scroll down in the documentation you will find a similar example that looks like

`checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}`